lambdas/service/app.py

```diff
@@ -224,7 +224,67 @@ def generate_health_object(_event: chalice.app.CloudWatchEvent):
     controller.generate_cache()
 
 
-@app.route('/version', methods=['GET'], cors=True)
+version_spec = {
```

Comment:
This should be inlined so the spec textually follows the `@app.route` line. Extraction of specs into variables should only be done for reuse.